src/System.AppContext/ref/System.AppContext.csproj

```diff
@@ -1,6 +1,9 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
+  <PropertyGroup Condition=" '$(Configuration)' == 'Debug' ">
+    <CheckForOverflowUnderflow>true</CheckForOverflowUnderflow>
```

Comment:
Do we have a feel for how bad it is? If it isn't bad I would prefer we go ahead and do it for everything. 

If we do feel we need to do it incrementally we should probably set some other single opt-in property in these projects and set it in the common targets based on that property. 

If we do keep it in the individual projects then we should condition it on the ConfigurationGroup property instead of Configuration.